libr/io/io.c

```diff
@@ -493,20 +414,23 @@ R_API bool r_io_read_at_mapped(RIO *io, ut64 addr, ut8 *buf, int len) {
 		memset(buf, io->Oxff, len);
 	}
 	if (io->va) {
-		ret = on_map_skyline(io, addr, buf, len, R_IO_READ, fd_read_at_wrap, false);
+		ret = on_map_skyline(io, addr, buf, len, R_IO_READ, fd_read_at_wrap, NULL);
```

Comment:
Missing space bwfore (.. and if u killed skyline why name this function like this?

Any perf related comments?

Reply:
did I kill the skyline? I just went back to a state, where map priorization works, and then tried to make it build again